SimpleFSDM-SET_COMMAND/commands/set.py

```diff
@@ -0,0 +1,25 @@
+from Icommand import *
+from model.database import *
+from response.exceptions import *
+
+
+class SetCommand(ICommand):
+    def __init__(self, database_name, table_name=None, data=None):
+        self.data = SetCommand.validate(database_name, data)
+        self.database_name = database_name
+        self.table_name = table_name
+
+    def execute(self):
+        database = Database(database_name=self.database_name)

```

Comment:
`Database` class takes database_object there is no such argument `database_name`